### ISSUE TITLE
Version conflict between PyPi and the requirements according to github

### ISSUE THREAD

**Issue by VaghehDashti** [radinhamidi/Team_Formation_Library#1]
I installed the PyTFL package from PyPi (pip install pytfl) and noticed that there is a distinction between the packages being installed from the ones mentioned on the README. For instance, tensorflow 2.6 and keras 2.6 are installed instead of tensorflow 1.15 and keras 2.0. Also, it does not install matplotlib and tensorflow_probability. Finally, importing pickle in the /dal/load_dblp_data.py is missing. Here is the list of packages that I am running the code with:

Name                    Version                   Build  Channel
absl-py                   0.13.0                   pypi_0    pypi
astunparse                1.6.3                    pypi_0    pypi
ca-certificates           2021.7.5             haa95532_1
cachetools                4.2.2                    pypi_0    pypi
certifi                   2021.5.30        py38haa95532_0
charset-normalizer        2.0.4                    pypi_0    pypi
clang                     5.0                      pypi_0    pypi
click                     8.0.1                    pypi_0    pypi
cloudpickle               1.6.0                    pypi_0    pypi
colorama                  0.4.4                    pypi_0    pypi
cycler                    0.10.0                   pypi_0    pypi
cython                    0.29.21                  pypi_0    pypi
decorator                 5.0.9                    pypi_0    pypi
dm-tree                   0.1.6                    pypi_0    pypi
flatbuffers               1.12                     pypi_0    pypi
gast                      0.4.0                    pypi_0    pypi
gensim                    4.0.1                    pypi_0    pypi
google-auth               1.35.0                   pypi_0    pypi
google-auth-oauthlib      0.4.5                    pypi_0    pypi
google-pasta              0.2.0                    pypi_0    pypi
grpcio                    1.39.0                   pypi_0    pypi
h5py                      3.1.0                    pypi_0    pypi
idna                      3.2                      pypi_0    pypi
joblib                    1.0.1                    pypi_0    pypi
keras                     2.6.0                    pypi_0    pypi
keras-preprocessing       1.1.2                    pypi_0    pypi
kiwisolver                1.3.1                    pypi_0    pypi
markdown                  3.3.4                    pypi_0    pypi
matplotlib                3.4.3                    pypi_0    pypi
nltk                      3.6.2                    pypi_0    pypi
numpy                     1.19.5                   pypi_0    pypi
oauthlib                  3.1.1                    pypi_0    pypi
openssl                   1.1.1k               h2bbff1b_0
opt-einsum                3.3.0                    pypi_0    pypi
pillow                    8.3.1                    pypi_0    pypi
pip                       21.0.1           py38haa95532_0
protobuf                  3.17.3                   pypi_0    pypi
pyasn1                    0.4.8                    pypi_0    pypi
pyasn1-modules            0.2.8                    pypi_0    pypi
pyparsing                 2.4.7                    pypi_0    pypi
pytfl                     1.0.0                    pypi_0    pypi
python                    3.8.11               h6244533_1
python-dateutil           2.8.2                    pypi_0    pypi
regex                     2021.8.3                 pypi_0    pypi
requests                  2.26.0                   pypi_0    pypi
requests-oauthlib         1.3.0                    pypi_0    pypi
rsa                       4.7.2                    pypi_0    pypi
scikit-learn              0.24.2                   pypi_0    pypi
scipy                     1.7.1                    pypi_0    pypi
setuptools                52.0.0           py38haa95532_0
six                       1.15.0                   pypi_0    pypi
sklearn                   0.0                      pypi_0    pypi
smart-open                5.2.0                    pypi_0    pypi
sqlite                    3.36.0               h2bbff1b_0
tensorboard               2.6.0                    pypi_0    pypi
tensorboard-data-server   0.6.1                    pypi_0    pypi
tensorboard-plugin-wit    1.8.0                    pypi_0    pypi
tensorflow                2.6.0                    pypi_0    pypi
tensorflow-estimator      2.6.0                    pypi_0    pypi
tensorflow-probability    0.13.0                   pypi_0    pypi
termcolor                 1.1.0                    pypi_0    pypi
threadpoolctl             2.2.0                    pypi_0    pypi
tqdm                      4.62.1                   pypi_0    pypi
typing-extensions         3.7.4.3                  pypi_0    pypi
urllib3                   1.26.6                   pypi_0    pypi
vc                        14.2                 h21ff451_1
vs2015_runtime            14.27.29016          h5e58377_2
werkzeug                  2.0.1                    pypi_0    pypi
wheel                     0.37.0             pyhd3eb1b0_0
wincertstore              0.2                      py38_0
wrapt                     1.12.1                   pypi_0    pypi

**Comment by radinhamidi** [comment]
Hi,

Thanks for the feedback. You are right, the dependencies list must be updated. I updated the list, please check now. 🙂
